test/addons_test.go

```diff
@@ -345,6 +493,7 @@ func testgroup(t *testing.T, groupname string, version string, jobs ...clusterTe
 		if err != nil {
 			if strings.Contains(err.Error(), "directory not found") {
 				t.Logf("no need to upgrade test %s, it appears to be a new addon (no previous revisions found in branch %s)", newAddon.GetName(), comRepoRef)
+				addonDeploymentsArray = append(addonDeploymentsArray, newAddon)
```

Comment:
if no upgrade test needed, we probably don't need to actually do anything more for the addon right? this would essentially rerun `testGroupDeployment` which is just deploying the latest addon version